modules/phase_field/src/userobjects/PolycrystalVoronoi.C

```diff
@@ -166,6 +209,59 @@ PolycrystalVoronoi::precomputeGrainStructure()
         _centerpoints[grain](2) = _bottom_left(2) + _range(2) * 0.5;
     }
   }
+
+  // Build a KDTree that is used to speedup point search
+  buildSearchTree();
+}
+
+void
+PolycrystalVoronoi::buildSearchTree()
+{
+  if (!_use_kdtree)
+    return;
+
+  auto midplane = _bottom_left + _range / 2.0;
+  dof_id_type lgrianid = 0;
```

Comment:
You misspelled this variable. Also, I'm not a huge fan of the name. It doesn't follow naming conventions very well.